### ISSUE TITLE
Incorrect door state value in README

### ISSUE THREAD

**Issue by pawlh** [ConnorTheFox/myq-node#4]
The README states that a returned door state of 9 means the door is open, although it appears that 1 is the actual value returned for open.

(I have checked other MyQ projects, and  [this one ](https://github.com/thomasmunduchira/myq-api/blob/master/readme.md) defines 9 simply as "not closed"; I am unsure of the meaning, but as of yet I have not seen 9 as a returned value for any circumstance)

**Comment by JustAnOpossum** [comment]
Huh, that's strange. I just pulled the values from the HTTP request. I'll check again in the next couple of days to verify. Thank you for bringing this to my attention.

**Comment by pawlh** [comment]
Yeah, I'm not quite sure what's up. Let me know if you find anything concurring or refuting it; I've tried to find a circumstance that returns 9 without avail, and I don't want to get caught off guard with an unexpected value. Thanks for your work!
